php/class-post-type.php

```diff
@@ -380,18 +386,28 @@ public function render_data_metabox( $post ) {
 				}
 				echo '</span>';
 			}
-
+			if ( isset( $conflicts_settings[ $setting_id ] ) ) {
+				$setting_id_key = trim( str_replace( array( '][', '[', ']' ), '-', $setting_id ), '-' );
+				$title_text = sprintf( esc_html__( '%s has potential Snapshot conflicts (click to expand)', 'customize-snapshots' ), $setting_id );
+				echo '<a href="#TB_inline?width=600&height=550&inlineId=snapshot-' . esc_attr( $setting_id_key ) . '" class="dashicons dashicons-warning thickbox snapshot-thickbox" title="' . $title_text . '"></a>'; ?>
+				<div id="snapshot-<?php echo esc_attr( $setting_id_key ); ?>" style="display:none;">
+					<?php foreach ( $conflicts_settings[ $setting_id ] as $data ) { ?>
+						<details>
+							<summary>
+								<code><?php echo $data['name'] ?></code>
+								<a href="<?php echo esc_url( $data['editLink'] ); ?>">
+									(<?php _e( 'edit', 'customize-snapshots' ); ?>)
```

Comment:
Best to use `esc_html_e()` here instead of `_e()`. Also, I think the parens should be included in the translated string.